environment/src/main/resources/schema/app/20200324204731_CB-6122_Environment_level_http_proxy_config.sql

```diff
@@ -0,0 +1,14 @@
+-- // CB-6122 Environment level http proxy config
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE environment ADD COLUMN IF NOT EXISTS proxyconfig_id BIGINT NULL;
```

Comment:
if we add the id to proxyconfig we won't increase the env table size, and we could move easily to assign multiple proxy to one env if needed

Reply:
@lacikaaa I dont think that we would like to support multiple proxy on one env. We discussed this with @keyki and we do not see any valid usecase.